content/applications/hr/payroll/contracts.rst

```diff
@@ -0,0 +1,249 @@
+
+Enter the amount in the various fields, or check a box to apply a benefit. Some options that can be
+entered here include :guilabel:`Meal Vouchers`, :guilabel:`Fuel Card`, :guilabel:`Internet`,
+:guilabel:`Paid Time Off`, etc.
+
+Some fields may be automatically filled in based off of the contracts selected in the
+:guilabel:`Contract Details` tab.
+
+Attachment of Salary
+--------------------
+
+.. image:: contracts/salary-attachment.png
+   :align: center
+   :alt: Optional tabs for a new contract.
+
+Any automatic deductions or allocations for an employee, such as child support payments and wage
+garnishments, are referred to as a *salary attachment*. This section is where all of these
+deductions or allocations are set.
+
+To add a new deduction, click on :guilabel:`Add a line`. Type in a description for the allocation
+under :guilabel:`Description`.
+
+.. image:: contracts/garnishment.png
+   :align: center
+   :alt: Enter a new line for each type of garnishment.
+
+Select the :guilabel:`Garnished Type` from the drop-down menu. Chose from:
+
+- :guilabel:`Assignment of Salary` - Any payments taken out towards something that is *not* child
+  support. Typically any garnishments such as lawsuit payments, payments toward taxes owed, etc.
+- :guilabel:`Allocaiton of Salary` - Any deduction that is not required but voluntary, such as a
+  pre-tax allocation to a college savings account.
+- :guilabel:`Child Support` - Any payments taken out specifically for child support.
+
+Enter the start and end dates the entry applies to by clicking on the drop-down menu under
+:guilabel:`From` and :guilabel:`To`, navigating to the correct month and year by using the arrow
+icons, then clicking on the date.
+
+Last, enter the :guilabel:`amount` that each payslip will pay towards the entry.
+
+Delete a line by clicking on the trash can icon at the end of the line.
+
+.. image:: contracts/delete-garnishment.png
+   :align: center
+   :alt: Remove a new line by clicking the delete icon.
+
+Save and send the contract
+--------------------------
+
+Once contracts have been created and/or modified, save the contract by clicking the :guilabel:`Save`
+smart button. Next, the contract must be sent to the employee to be signed.
+
+Click on one of the following smart buttons:
+
+.. image:: contracts/send-contract.png
+   :align: center
+   :alt: Send the contract to the employee via one of the smart buttons.
+
+- :guilabel:`Generate Simulation Link`: This option is for Belgian companies only. Clicking this
+  will open a pop-up window that contains the basic information from the contract as well as a link
+  for the contract when using the salary configurator. Click :guilabel:`Send Offer` to send an email
+  to the employee so they can sign the contract.
+
+.. note::
+   In order to send a contract using the :guilabel:`Generate Simulation Link`, there must be a
+   signature field in the contract PDF being sent to the employee so they can sign it.
```

Comment:
I would move this note to be right below the image that is currently below it (simulation.png). As it is now, I kind of expect the image to illustrate the note since it comes after it.